corehq/apps/reports/templates/reports/tableau_template.html

```diff
@@ -2,18 +2,29 @@
 {% load compress %}
 {% load hq_shared_tags %}
 {% load i18n %}
+{% load static %}
 
 {% block page_content %}
 
 {% initial_page_data "server_type" server_type %}
 {% initial_page_data "ticket" ticket %}
 {% initial_page_data "validate_hostname" validate_hostname %}
+{% initial_page_data "server_address" server_address %}
+{% initial_page_data "target_site" target_site %}
+{% initial_page_data "domain_username" domain_username %}
 {% initial_page_data "view_url" view_url %}
+{% registerurl 'tableau_visualisation_ajax' request.domain %}
```

Comment:
`tableau_visualisation_ajax` => `tableau_visualization_ajax `

Reply:
(I feel like your trip to London is is coming out here...also sometimes I wish we had just called these "reports".)